config/routes/dev/framework.yaml

```diff
@@ -0,0 +1,3 @@
+_errors:
+    resource: '@FrameworkBundle/Resources/config/routing/errors.xml'
+    prefix: /__error
```

Comment:
Why not `/_error` as before?

Reply:
That was my faults, I thought both routes could live together during the upgrade path if you force the installation of the 4.4 recipe. But that does makes no sense anymore after your changes.